scipy/optimize/_lsq/least_squares.py

```diff
@@ -234,45 +234,40 @@ def least_squares(
         kwargs={}):
     """Solve a nonlinear least-squares problem with bounds on the variables.
 
-    Let f(x) be an m-dimensional function of n variables and rho(s) be a
-    scalar function, `least_squares` finds a local minimum of::
+    Given f(x), an m-dimensional function of n variables (the residuals),
+    and rho(s), a scalar function (the loss function), `least_squares`
+    finds a local minimum of the cost function F(x)::
 
         F(x) = 0.5 * sum(rho(f_i(x)**2), i = 0, ..., m - 1)
         subject to lb <= x <= ub
 
-    We call f(x) as a vector of residuals or simply residuals, and F(x) as a
-    cost function or simply cost.
-
-    We call rho(s) as a loss function, its purpose to reduce the influence
-    of outliers on the solution.
-
-    Partial derivatives of f with respect to x form m-by-n matrix called
-    Jacobian, where an element (i, j) equals the partial derivative of f[i]
-    with respect to x[j].
+    The purpose of the loss function rho(s) is to reduce the influence of
+    outliers on the solution.
 
     Parameters
     ----------
     fun : callable
-        Function which computes the vector of residuals with the signature
+        Function which computes the vector of residuals, with the signature
         ``fun(x, *args, **kwargs)``, i.e., the minimization proceeds with
-        respect to it's first argument. The argument ``x`` passed to this
-        function is ndarray of shape (n,) (never a scalar, even for n=1).
+        respect to its first argument. The argument ``x`` passed to this
+        function is an ndarray of shape (n,) (never a scalar, even for n=1).
         It must return a 1-d array_like of shape (m,) or a scalar.
     x0 : array_like with shape (n,) or float
         Initial guess on independent variables. If float, it will be treated
         as a 1-d array with one element.
     jac : {'2-point', '3-point', 'cs', callable}, optional
-        Method of computing the Jacobian matrix. The keywords select the
+        Method of computing the Jacobian matrix. The keywords select a
```

Comment:
I think it's better to give a definition of Jacobian here, not on the very last sentence.